src/rred_reports/reports/filler.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+
+import pandas as pd
+from docx import Document
+from docx.table import Table
+
+
+class TemplateFillerException(Exception):
+    """Custom exception generator for the TemplateFiller class"""
+
+    def __init__(self, message: str):
+        super().__init__(message)
+        self.message = message
+
+    def __repr__(self) -> str:
+        return self.message
+
+
+class TemplateFiller:
+    """TemplateFiller class to modify tables in .docx files.
+
+    Provides save/loading, table modification and input verification.
+    """
+
+    def __init__(self, template_path: Path):
+        self.doc = Document(template_path)
+        self.tables = self.doc.tables
+
+    @staticmethod
+    def remove_all_rows(table: Table):
+        """Remove all rows in a table except for title
+
+        Args:
+            table (Table): Table within a .docx file
+        """
+        total_rows = len(table.rows) - 1
+        for row_number in range(total_rows, 0, -1):
+            row = table.rows[row_number]
+            row._element.getparent().remove(row._element)  # pylint: disable=protected-access
```

Comment:
yeah I couldn't find a public way to do it, and it seems the most sane way to approach it